packages/web/src/components/TeeForm.vue

```diff
@@ -226,6 +250,10 @@ const analytics = analyticsStore()
 // const usedTracks: UsedTrack[] | any[] = tracks.getAllUsedTracks
 const trackValues: any[] = tracks.getAllUsedTracksValues
 
+// @ts-ignore
+const metaEnv = import.meta.env
+const contactEmail = ref<string>(metaEnv.VITE_BREVO_SENDER_EMAIL || 'france-transition@beta.gouv.fr')
```

Comment:
@yvalentin 

suggestion: to replace existing line, WDYT ?

```suggestion
const contactEmail = metaEnv.VITE_BREVO_SENDER_EMAIL || 'france-transition@beta.gouv.fr'
```

Reply:
yes but can you type the constant `contactEmail`

```js
const contactEmail: string = metaEnv.VITE_BREVO_SENDER_EMAIL || 'france-transition@beta.gouv.fr'
```

Reply:
suggestion: @JulienParis And maybe, we need to change the name of `VITE_BREVO_SENDER_EMAIL` to something like: `VITE_CONTACT_EMAIL` ?

Reply:
note: changed in [this commit](https://github.com/betagouv/transition-ecologique-entreprises-widget/pull/135/commits/3df4c62f34668a1ec06d8c70fd3b19ed8acf664e)